tinyagentos/routes/peer.py

```
@@ -338,3 +346,175 @@ async def peer_ack(body: PeerAck, request: Request):
+                member_kind="human",
+                role="member",
+            )
+            await project_store.log_activity(
+                project_id, contact_id, "member.added",
+                {"member_id": contact_id, "kind": "human", "via": "collab_invite"},
+            )
+            logger.info(
+                "peer_inbox: %s → added %s as human member to project %s",
+                kind, contact_id, project_id,
+            )
+        except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)